lib/withTypescript.js

```diff
@@ -9,7 +9,20 @@ module.exports = (base) => ({
 
     '@typescript-eslint/array-type': ['error', { default: 'generic' }],
     '@typescript-eslint/prefer-optional-chain': ['error'],
-
+    '@typescript-eslint/consistent-type-definitions': ['error', 'interface'],
+    '@typescript-eslint/consistent-type-imports': 'error',
+    '@typescript-eslint/no-floating-promises': 'error',
+    '@typescript-eslint/no-misused-promises': [
+      'error',
+      {
+        'checksVoidReturn': false,
+      },
+    ],
```

Comment:
Why disable?

Reply:
I actually can't remember why I added this... maybe I saw it throwing an error on one of our projects that I was testing on and thought we better keep it in, but now I think about it better just to have the error and fix it.